static/templates/edit_bot.hbs

```diff
@@ -1,36 +1,45 @@
+<div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
+    <h3 class="inline-block" id="change_full_name_modal_label">{{t "Edit bot" }}</h3>
+    <div class="alert-notification change_full_name_info"></div>
+</div>
 <form class="edit_bot_form form-horizontal" data-target="edit-bot" data-title="{{t 'Edit bot' }}" data-user-id="{{bot.user_id}}"
   data-email="{{bot.email}}" data-type="{{bot.bot_type}}">
     <div class="bot_edit_errors alert alert-error hide"></div>
-    <div class="">
-        <label>{{t "Bot email" }}</label>
-        <div class="edit_bot_email">{{bot.email}}</div>
-    </div>
-    <div class="edit-bot-form-box">
+    <div class="edit_bot_form_container">
         <div class="">
-            <label for="edit_bot_name">{{t "Full name" }}</label>
-            <input id="edit_bot_name" type="text" name="bot_name" class="edit_bot_name required" value="{{bot.full_name}}" maxlength=50 />
-            <div><label for="edit_bot_name" generated="true" class="text-error"></label></div>
+            <label>{{t "Bot email" }}</label>
+            <div class="edit_bot_email">{{bot.email}}</div>
         </div>
-        <div class="edit-bot-owner">
-            <label for="bot_owner_select">{{t "Owner" }}</label>
-            <div class="select-form">
-                {{> bot_owner_select }}
+        <div class="edit-bot-form-box">
+            <div class="">
+                <label for="edit_bot_name">{{t "Full name" }}</label>
+                <input id="edit_bot_name" type="text" name="bot_name" class="edit_bot_name required" value="{{bot.full_name}}" maxlength=50 />
+                <div><label for="edit_bot_name" generated="true" class="text-error"></label></div>
+            </div>
+            <div class="edit-bot-owner">
+                <label for="bot_owner_select">{{t "Owner" }}</label>
+                <div class="select-form">
+                    {{> bot_owner_select }}
+                </div>
+            </div>
+            <div id="service_data">
+            </div>
+            <div class="avatar-section">
+                <label for="bot_avatar_file_input">Avatar</label>
+                <input type="file" name="bot_avatar_file_input" class="notvisible edit_bot_avatar_file_input" value="{{t 'Upload profile picture' }}" />
+                <div class="edit_bot_avatar_file"></div>
+                <button type="button" class="btn btn-default edit_bot_avatar_upload_button">{{t "Choose avatar" }}</button>
```

Comment:
![Screenshot_1](https://user-images.githubusercontent.com/25329595/77557305-9f718200-6edf-11ea-9bda-99c4dda41f05.png)

The `Choose avatar` button has sharp corners. I think we want it to be the same as (or similar to) the original button.

Reply:
ok sure